python/cugraph/cugraph/structure/symmetrize.py

```diff
@@ -298,7 +298,7 @@ def symmetrize(
             return (
                 output_df[source_col_name],
                 output_df[dest_col_name],
-                output_df[value_col.columns],
+                output_df[list(value_col.columns)],
```

Comment:
TODO: This may be a dask-expr bug? Column projection using anything other than a list seems fragile.

Reply:
I'm not observing this bug locally anymore, but I'd still like to keep this precaution in place.

Reply:
Should we leave a comment? Is it worth raising a tracking issue on cuGraph for follow up?

Reply:
I'll be honest: I don't actually think this "fix" is required, because removing it doesn't seem to cause test failures for me locally (was probably specific to an earlier combination of dask/dask-expr/dask-cudf). However, I left it for now because it will take a long time for "real CI" to tell me that it actually is a problem.

With that said, I'll be happy to give it a try now that I'm realizing it's only cudf/dask-cudf that is about to freeze (and not cugraph).